### ISSUE TITLE
[Bug] Check if you have the latest updates

### ISSUE THREAD

**Issue by madereddy** [MrLoLf/HoneygainAutoClaim#16]
I forked the repo, set the variables, and updated both manual and daily to 0 for IsJWT. When running the code says it needs to be updated when the repo is two commits ahead of your repo.

```
Your repo is not up-to-date with the original repo
Please update your repo to the latest commit to get new updates and bug fixes
```
https://github.com/madereddy/HoneygainAutoClaim/actions/runs/7398327207/job/20127810813

**Comment by MrLoLf** [comment]
Yeah that's done, because the update checks not any version to update,  rather commit changes. This is kinda bad, and I thought of a different solution where I would have a dev branch with all the changes and a main branch with the latest release which would have tags I can check for. But this requires more work and maybe there is a way for automating this. Thank you for opening this issue.

**Comment by gorouflex** [comment]
```

if os.getenv('GITHUB_ACTIONS') == 'true':
    user_repo = os.getenv('GITHUB_REPOSITORY')
    original_repo = 'MrLoLf/HoneygainAutoClaim'
    user_url = f'https://api.github.com/repos/{user_repo}/commits?path=main.py'
    original_url = f'https://api.github.com/repos/{original_repo}/commits?path=main.py'

```

This may fix this issue, since it only checked the latest commit for the `main.py` file. At first, I created this feature to check for updates in the whole repo, since my repo was not completed at that time
@MrLoLf , @madereddy

**Comment by gorouflex** [comment]
Should be closed after the #18
